dapps/marketplace/src/pages/App.js

```diff
@@ -136,6 +145,11 @@ class App extends Component {
             <Route path="/about/payments" component={AboutPayments} />
             <Route path="/about/tokens" component={AboutToken} />
             <Route exact path="/campaigns" component={GrowthCampaigns} />
+            <Route
```

Comment:
Would `<Route exact path="/campaigns/:navigation" component={GrowthCampaigns} />` work?

Reply:
Yup thanks. No idea why I complicated this part

Reply:
I don't see this change yet

Reply:
This is so weird. Doesn't show up correctly in the diff. But if you check the source file the change is there: 
https://github.com/OriginProtocol/origin/blob/sparrowDom/mobileFirstRewards/dapps/marketplace/src/pages/App.js#L148-L152